src/main/java/net/minecraftforge/fml/common/network/FMLEventChannel.java

```diff
@@ -177,6 +177,19 @@ public void sendToAllAround(FMLProxyPacket pkt, NetworkRegistry.TargetPoint poin
         channels.get(Side.SERVER).writeAndFlush(pkt).addListener(ChannelFutureListener.FIRE_EXCEPTION_ON_FAILURE);
     }
 
+    /**
+     * Send to all tracking the point
+     * The {@code range} field of the {@link NetworkRegistry.TargetPoint} is ignored.
```

Comment:
would like feedback about this. should TargetPoint.range be ignored or be considered (so that the outbound target has *two* constraints, distance and tracking).

Reply:
I think range should be ignored. The player is either tracking the chunk or not. If they're not, I think it doesn't really matter if they're in range because they will not have use for data in a chunk they're not tracking.